layout/_partial/header.ejs

```diff
@@ -3,12 +3,16 @@
   <% if (theme.logo && theme.logo.enabled) { %>
     <% if (theme.gravatar && (theme.gravatar.email || theme.gravatar.hash) && theme.logo.gravatar) { %>
       <% if (theme.gravatar.email) { %>
-        <img id="logo" class="u-logo" src="<%- gravatar(theme.gravatar.email) %>" />
+        <img id="logo" alt class="u-logo" 
+          srcset="<%- gravatar(theme.gravatar.email) %>?s=<%= theme.logo.width %>, <%- gravatar(theme.gravatar.email) %>?s=<%= theme.logo.width*1.5 %> 1.5x, <%- gravatar(theme.gravatar.email) %>?s=<%= theme.logo.width*2 %> 2x"
```

Comment:
Replacing CSS background-image with an `img` tag had a negative impact on Lighthouse Accessibility score. 
I added an empty `alt` attribute to restore previous behaviour (image is purely decorative)

While looking at lighthouse report, I added the `srcset` combined with gravatar size parameter adds support for high-density screens (Fixes "Ensure images display with the correct aspect ratio" rules in Lighthouse _Best Practice_ report.